src/features/import/components/ImportDialog/Configure/index.tsx

```diff
@@ -26,63 +26,90 @@ const Configure: FC<ConfigureProps> = ({ onClose, onRestart, onValidate }) => {
     number | null
   >(null);
   const { orgId } = useNumericRouteParams();
+  const theme = useTheme();
   const { forwardMessageDisabled, numRows, uiDataColumns } =
     useUIDataColumns(orgId);
+  const [loading, setLoading] = useState(false);
   const getPreflightStats = useConfigure(orgId);
 
   return (
-    <Box display="flex" flexDirection="column" height="100%" overflow="hidden">
-      <ImportHeader activeStep={ImportStep.CONFIGURE} onClose={onClose} />
-      <Box display="flex" flexGrow={1} overflow="hidden">
+    <>
+      {loading && (
         <Box
+          alignItems="center"
           display="flex"
           flexDirection="column"
-          sx={{ overflowY: 'auto' }}
-          width="50%"
+          justifyContent="center"
+          paddingY={4}
         >
-          <SheetSettings
-            clearConfiguration={() => setColumnIndexBeingConfigured(null)}
-          />
-          <Mapping
-            clearConfiguration={() => setColumnIndexBeingConfigured(null)}
-            columnIndexBeingConfigured={columnIndexBeingConfigured}
-            columns={uiDataColumns}
-            onConfigureStart={(columnIndex: number) =>
-              setColumnIndexBeingConfigured(columnIndex)
-            }
-          />
+          <CircularProgress sx={{ color: theme.palette.statusColors.blue }} />
+          <Typography sx={{ color: theme.palette.text.primary }}>
+            {messages.importStatus.loadingState()}
```

Comment:
This text says "Importing", so maybe let's use another one here where it's not really importing, just validating the import? I would suggest that it says "Validating" or perhaps "Checking the configuration" instead. What do you think, text maestro @ziggabyte?

Reply:
I think "Validating" makes sense, because the user just pushed a button that says "Validate"!